test_login.py

```diff
@@ -1,9 +1,8 @@
 import pytest
-
 from pages.desktop.home import Home
 
 
-
+@pytest.mark.skip
```

Comment:
Can we add a reason here, just so we remember why we skipped. https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions